acoustid/future/fpindex/client.py

```
@@ -0,0 +1,278 @@
+        kwargs: dict[str, Any] = {
+            "timeout": aiohttp.ClientTimeout(total=timeout or self.timeout),
+        }
+        if data is not None:
+            kwargs["json"] = data
+
+        async with self.session.request(method, url, **kwargs) as response:
+            status = response.status
+
+            if expected_status and status not in expected_status:
+                body = await response.text()
+                logger.error(f"Unexpected status {status} for {method} {url}: {body}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)